apps/aws-notebook/samples/ecs/fargate/vpn/create_app.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+
+[[ -z "${AWS_ACCESS_KEY_ID}" ]] && echo "AWS_ACCESS_KEY_ID is nil" 1>&2 && exit 1
+
+[[ -z "${AWS_SECRET_ACCESS_KEY}" ]] && echo "AWS_SECRET_ACCESS_KEY is nil" 1>&2 && exit 1
+[[ -z "${AWS_SESSION_TOKEN}" ]] && echo "AWS_SESSION_TOKEN is nil" 1>&2 && exit 1
+[[ -z "${AWS_REGION}" ]] && echo "AWS_REGION is nil" 1>&2 && exit 1
+[[ -z "${TASK_IMAGE}" ]] && echo "TASK_IMAGE is nil, please provide the url of notebook image,you could build it from this dockerfile:https://github.com/crafting-demo/solutions/blob/master/apps/aws-notebook/image/Dockerfile and push it to your aws container registry" 1>&2 && exit 1
+[[ -z "${CRAFTING_ORG}" ]] && echo "CRAFTING_ORG is nil, please provide crafting orignization name" 1>&2 && exit 1
+
+
+[[ $# -lt 1 ]] && echo "Missing APP_NAME. Usage: create_app.sh APP_NAME" && exit 1
+
+
+APP_NAME="$1"
+ECS_CLUSTER_NAME=$(terraform output -raw ecs_cluster_name)
+SUBNET_ID=$(terraform output -raw subnet_id)
+SERVICE_LAUNCH_TYPE=$(terraform output -raw service_launch_type)
+SECURITY_GROUP=$(terraform output -raw security_group)
+
+# Prepare snapshot
+echo "🌸 Preparing snapshot for sandbox"
+cs app create tmpdemo-notebook -O ${CRAFTING_ORG} - << EOF
+workspaces:
+  - name: dev
+    checkouts:
+      - path: solutions
+        repo:
+          git: https://github.com/crafting-demo/solutions.git
+        version_spec: master
+
+EOF
+cs sandbox create tmpdemo-notebook -a tmpdemo-notebook -O ${CRAFTING_ORG}
+cs ssh -W tmpdemo-notebook/dev -O ${CRAFTING_ORG} bash -c "cd solutions/shared/snapshots/notebook/ && bash build_base.sh base-notebook-v1"
+cs sandbox delete tmpdemo-notebook -O ${CRAFTING_ORG} --force
+cs app delete tmpdemo-notebook -O ${CRAFTING_ORG} --force
+
```

Comment:
Needs better error handling. If any error happens, with `set -e`, the script will abort. Do we want to cleanup the sandbox?